pom.xml

```diff
@@ -308,6 +307,7 @@
         <module>web-common</module>
         <module>servlet-dist</module>
         <module>servlet-feature-pack</module>
+        <module>feature-pack</module>
```

Comment:
that shouldn't matter at all. as long as there are proper dependencies in place, maven will figure out the proper order.